packages/cli/internal/pkg/cli/context/manager.go

```diff
@@ -120,6 +120,7 @@ func NewManager(profile string) *Manager {
 		baseProps: baseProps{homeDir: homeDir},
 		imageRefs: environment.CommonImages,
 		ecrClient: aws.EcrClient(profile),
+		region:    aws.Region(profile),
```

Comment:
Just to double check, when the profile is default (i.e. not passed through), does the region used the one under default/default to us-east-1 if its not set in ~/.aws/profile

Reply:
Yes, it takes the default region if none is provided. 
aws.Region calls getProfileConfig which fetches for the default.

* https://github.com/aws/amazon-genomics-cli/blob/a1ba41f9085af37fe2f69b65a26ac846ddf92c27/packages/cli/internal/pkg/aws/provider.go#L134-L138
* https://github.com/aws/amazon-genomics-cli/blob/a1ba41f9085af37fe2f69b65a26ac846ddf92c27/packages/cli/internal/pkg/aws/provider.go#L146-L156